README.md

```diff
@@ -176,41 +174,71 @@ This option is useful for "blacklisting" a transform like `transform-regenerator
 
 ### `useBuiltIns`
 
-`boolean`, defaults to `false`.
-
-A way to apply `babel-preset-env` for polyfills (via "babel-polyfill").
+`boolean`, defaults to `true`.
 
-> NOTE: This does not currently polyfill experimental/stage-x built-ins like the regular "babel-polyfill" does.
-> This will only work with npm >= 3 (which should be used with Babel 6 anyway)
+A way to apply `babel-preset-env` for polyfills (via `babel-polyfill`).
 
-```
+```sh
 npm install babel-polyfill --save
 ```
 
-This option enables a new plugin that replaces the statement `import "babel-polyfill"` or `require("babel-polyfill")` with individual requires for `babel-polyfill` based on environment.
+#### `useBuiltIns: true`
```

Comment:
Are you missing docs for useBuiltIns: false now? May seem obvious, but now that it's not the default you should at least describe the option.